oxidation/libparsec/crates/platform_device_loader/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "libparsec_platform_device_loader"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["use-rustcrypto"]
+use-rustcrypto = ["libparsec_crypto/use-rustcrypto"]
+use-sodiumoxide = ["libparsec_crypto/use-sodiumoxide"]
+
+[dependencies]
+libparsec_client_types = { path = "../client_types", default-features = false }
+libparsec_crypto = { path = "../crypto", default-features = false }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+base64 = "0.21.0"
+web-sys = { version = "0.3.60", features = ["Window", "Storage"] }
+
+[dev-dependencies]
+tests_fixtures = { path = "../tests_fixtures" }
+
+async-std = { version = "1.12.0", features = ["attributes"] }
```

Comment:
What is this used for ?

Reply:
async rstest

Reply:
you could use tokio directly also ;)

Reply:
Can you add fixture to tokio ?

Reply:
```rust
    #[rstest]
    #[case(false, false)]
    #[case(false, true)]
    #[case(true, false)]
    #[case(true, true)]
    #[tokio::test]
    async fn test_chunk_clearing(
        #[future] alice_workspace_storage: TmpWorkspaceStorage,
        #[case] cache_only: bool,
        #[case] clear_manifest: bool,
    ) {
```

Reply:
Is there a preference ?

Reply:
:shrug:

Reply:
I'd rather use tokio given it's what we use in production

And I still don't get why we need to specify `async-std` as part of our own dependencies if it is a `rstest` dependency 🤔

Reply:
Because they don't provide the feature, so we set manually the feature:
See
```
To use this feature, you need to enable attributes in the async-std features list in your Cargo.toml:

async-std = { version = "1.5", features = ["attributes"] }
```

Reply:
Done